sample/src/main/java/io/getstream/chat/example/LoginActivity.java

```diff
@@ -0,0 +1,52 @@
+package io.getstream.chat.example;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+import android.widget.Toast;
+
+
+import io.getstream.chat.example.adapter.UserListItemAdapter;
+import io.getstream.chat.example.utils.UserStorage;
+
+public class LoginActivity extends AppCompatActivity {
```

Comment:
There should be also header with subtitle from `MainActivity`. 
It's convenient to know which SDK version we're using from login screen.